<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementForm</class>
 <widget class="QWidget" name="MeasurementForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="plotForm" name="channel1Widget" native="true"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="plotForm" name="channel2Widget" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="measurementPushButton">
     <property name="text">
      <string>Measure</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="stopPushButton">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="clearPushButton">
     <property name="text">
      <string>Clear all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotForm</class>
   <extends>QWidget</extends>
   <header>plotform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
